src/WebJobs.Extensions.DurableTask/Bindings/EntityTriggerAttributeBindingProvider.cs

```diff
@@ -53,6 +53,8 @@ public Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext contex
                 name = method.GetCustomAttribute<FunctionNameAttribute>()?.Name ?? method.Name;
             }
 
+            name = name.ToLowerInvariant();
```

Comment:
I think my question from the first review iteration got missed, but why do we need to lowercase `name` here? I was expecting that we didn't need to do this since we already treat function names as case-insensitive.

Reply:
Some of the tests were failing without that line, so it definitely mattered somewhere... Figuring out why might be a good idea, and adding a comment to explain why.